src/ui/components/filters/daterangefiltercomponent.js

```diff
@@ -74,8 +74,8 @@ export default class DateRangeFilterComponent extends Component {
      * @private
      */
     this._date = {
-      min: minDate || config.initialMin || todayString,
-      max: maxDate || config.initialMax || todayString
+      min: [minDate, config.initialMin, todayString].find(v => v !== undefined),
```

Comment:
I know this case is probably very very rare: Do we care that if `minDate` were null, we set the date to null? Whereas before it would go to the next fallback?

Reply:
yes that is intentional!

Reply:
minDate and maxDate will also be going away after this PR, since we're going to change how the min/max are persisted to use the PERSISTED_FILTER storage key